app/assets/javascripts/search.js.coffee.erb

```diff
@@ -0,0 +1,28 @@
+class @Search
+
+  constructor: ->
+    @bindEvents()
+
+  bindEvents: ->
+    # not that it's sane to send a request on each keystroke
+    # just a mvp
+    $("#search-input").keyup (e) =>
```

Comment:
So, I like the fact that this is a class, but this needs to be made unobtrusive and also uncoupled javascript.  Referencing global dom IDs in your coffee script classes is a big no no.